backend/schemas/scripts/generate_lookup_schemas.py

```diff
@@ -70,5 +70,5 @@ def process_cluster_names(arg):
             print("Unknown filename, exiting")
             sys.exit(1)
 
-        with open(sys.argv[2], "w") as write_file:
+        with open(sys.argv[2], "w", newline="\n") as write_file:
```

Comment:
Windows and unix have different philosophies about newlines. This can cause weird diffs that don't appear to show any actual differences between commits. Let's...commit...to one newline type.